app/components/ClientTaskList/TaskItem/TaskItem.tsx

```diff
@@ -22,13 +22,14 @@ const TaskItem = ({ task }: { task: TaskWithTags }): React.ReactElement => {
   const [isDueDateModalOpen, setIsDueDateModalOpen] = useState(false);
   const { deleteTask, toggleComplete } = useTaskStore();
   const { selectedTaskIds, setSelectedTaskIds } = useSelectedTaskStore();
-
   const { isLinking } = useNoteStore();
 
-  // get the background color for the task from the color col
-  const bgColor = task.color
-    ? COLORS.find((color) => color.name === task.color)?.class
-    : 'bg-note';
+  const taskTagColor =
+    task.taskTags.length > 0 ? task.taskTags[0].tag.color : 'green';
+
+  const bgColor = COLORS.find(
+    (color) => color.name === taskTagColor
+  )?.background;
```

Comment:
It might be nice to have an option to turn off the color coding - for those that it might be distracting for

Reply:
I'll add this as a ticket to my project